packages/project/package.json

```diff
@@ -28,7 +30,14 @@
     "tsup": "^6.1.3",
     "typescript": "4.7.4"
   },
-  "main": "lib/index.js",
+  "exports": {
+    ".": {
+      "import": "./lib/index.js",
+      "require": "./lib/cjs/index.cjs",
+      "default": "./lib/index.js"
+    },
+    "./server": "./server.js"
```

Comment:
q: why this doesnt have a require condition? 🤔

Reply:
Caz not used in tests. I added dual modules support only for tested code.